API.md

```diff
@@ -233,6 +233,21 @@ The file extension for the base64 string in [`document`](#document), if [`isBase
 />
 ```
 
+#### documentExtension
+string, Android only, required if the URL/path does not contain the file extension or if you're loading a Blob/File. Defaults to the extension in the [`document`](#document) prop. 
```

Comment:
not android only

Reply:
"if you're loading a Blob/File" unclear what this means? also applies to wrong extension

Reply:
Oops took that from web. Will fix.